airflow/providers/common/sql/operators/sql.py

```diff
@@ -622,8 +622,6 @@ def execute(self, context: Context):
         self.log.info("All tests have passed")
 
     def _generate_sql_query(self):
-        self.log.info("Partition clause: %s", self.partition_clause)
```

Comment:
logging might be important for someone so I think we should move the statement calling  `_generate_sql_query` outside __init__ if we can or keep it but make it debug, wdyt?

Reply:
I know I added that logging line while I was debugging some templating issues so it hasn't been around long. The function call itself needs to be in `__init__()` so the results gets templated correctly.  

This is also why I think the log line is redundant and can be safely removed: the templated `partition_clause` is available via the airflow UI so people can see it (or the lack of it).  I could see an argument for logging out the full query in the execute function for a "this is what ran" vs "this is what you can see through the UI", but the line this PR removes doesn't seem worthwhile.

Reply:
Maybe change the level to debug instead? Since you added it for debugging it could be useful again.

Reply:
Fine, I've amended the log level to debug.  I still don't think the line is needed, but at least it won't spam logs this way.